CLI tools/CurseForge to Packwiz-Modrinth.py

```diff
@@ -54,13 +58,32 @@ def extract_file(from_zip, from_file, to_path, from_desc, to_desc):
     if modrinth_overrides:
         os.system(packwiz_exe_path + " remove entityculling")
         os.system(packwiz_exe_path + " mr install entityculling")
-os.system(packwiz_exe_path + " refresh")
+        
+elif mmc_export_packwiz_export == False and refresh_only == True: 
+    os.system(packwiz_exe_path + " refresh")
 
 # Copy fresh manifest/modlist to git
 if is_legacy == False and refresh_only == False:
     extract_file(cf_zip_path, "manifest.json", git_path + "CurseForge", "CurseForge manifest.json", "Git")
     extract_file(cf_zip_path, "modlist.html", git_path + "CurseForge", "CurseForge modlist.html", "Git")
 
+# Export packwiz pack via mmc-export method
+
+if mmc_export_packwiz_export and not refresh_only:

```

Comment:
heh, here you used `not` but elsewhere `== False`

Reply:
I tried to follow your code style, but forgot for a second

Reply:
Hmm, btw why do you use `if mmc_export_modrinth_export: ...` instead of `if mmc_export_modrinth_export == True: ...`?

Reply:
Good point. I guess the reason why I used `== False` at all is that I forgot how to negate in Python 😄

Reply:
Lol. Anyway, is this an issue? I can change it to `== False`if needed

Reply:
Nah, I shall learn the Python negation 😄

Reply:
Heh, that's a good approach